src/services/api/base.ts

```diff
@@ -0,0 +1,45 @@
+import Vue from "vue";
+import axios, { AxiosInstance, AxiosResponse } from "axios";
+
+export class BaseApiService {
+
+  private baseUrl = process.env.VUE_APP_API_BASE_URL;
+  static _instance: BaseApiService;
+  http: AxiosInstance;
+  
+  constructor() {
+    this.http = axios.create({
+      baseURL: this.baseUrl,
+    });
+    this.http.defaults.headers.common = {
+      "Access-Control-Allow-Origin": "*",
+      "Access-Control-Allow-Methods": "POST, GET, PUT, OPTIONS, DELETE",
+      "Access-Control-Allow-Headers":
+        "Access-Control-Allow-Methods, Access-Control-Allow-Origin, Origin, Accept, Content-Type",
+      "Content-Type": "application/json",
+      Accept: "application/json",
+    };
+  }
+  static get instance() {
+    if (!this._instance) {
+      this._instance = new BaseApiService();
+    }
+    return this._instance;
+  }
+
+  read = async (resource: string = ""): Promise<AxiosResponse> => {
+    return await this.http.get(resource);
```

Comment:
Either embed this line in a try catch block or split this line into two. [ES-lint rule](https://github.com/eslint/eslint/blob/master/docs/rules/no-return-await.md) & [another example](https://jakearchibald.com/2017/await-vs-return-vs-return-await/#return-awaiting).

The same applies to all return await calls.